### ISSUE TITLE
v1.0.1604.0 issues

### ISSUE THREAD

**Issue by E66666666** [E66666666/GTAVManualTransmission#33]
What still works:
  * _Most_ patterns to find offsets.
  * Pattern to find "shift up" action.

What broke:
  * Turbo pattern.
  * Drive Force (was Next Gear offset `+ 0x28`, but should be `+ 0x2C` now (or `0x8A4`)).
  * Some fields in wheel data
  * Patterns for the rest of the gearbox patching
  * Patterns for steering patching

```md
[23:42:52.465] [ WARN] Turbo Offset: 0x0
[23:42:52.712] [DEBUG] PATCH: GEARBOX: Patching
[23:42:52.776] [ERROR] PATCH: GEARBOX: clutchLow patch failed
[23:42:52.840] [ERROR] PATCH: GEARBOX: clutchRevLimit patch failed
[23:42:52.907] [ERROR] PATCH: GEARBOX: shiftDown patch failed
[23:42:52.919] [DEBUG] PATCH: GEARBOX: Patched shiftUp  @ 0x00007FF6D37CCF56
[23:42:52.920] [ERROR] PATCH: GEARBOX: Patching failed
[23:44:29.178] [DEBUG] PATCH: STEERING: Patching
[23:44:29.257] [DEBUG] PATCH: Steering correction patch @ 0x0000000000000000
[23:44:29.258] [ERROR] PATCH: STEERING: Patch failed
```

Help would be nice, since I won't have any time during the holidays.

**Comment by E66666666** [comment]
Turbo and Drive Force are fixed in b810d860a7ad3942876d32ecd0b0c51fa24964b6, some wheel offsets I used are fixed in 63f46aba381aec8aed1f12ec2722c49480e18b0a due to a dumb mistake.

Still need new patching patterns.

**Comment by E66666666** [comment]
Fixed in f9ae9897df7ae2fed016cf3a81bc6a1fd18f83c8
